pkg/apis/testharness/v1beta1/test_types.go

```diff
@@ -66,7 +66,7 @@ type TestSuite struct {
 	// be removed it was created (unless --skipDelete is used).
 	Namespace string
```

Comment:
since you're already in here, would you mind adding the json mapping also for namespace? potentially all other things that does not have one?